KomodoPlatform/atomicDEX-Desktop
# Add assets from sidebar

Closes https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/2051

To test:
- launch app
- see new button in sidebar
- press the button
- see the coin activation menu
